packages/patternfly-4/react-core/src/helpers/constants.ts

```diff
@@ -1,3 +1,7 @@
-export const KEY_CODES = { ESCAPE_KEY: 27 };
+export const KEY_CODES = { ESCAPE_KEY: 27, TAB: 9, ENTER: 13 };
+
+export const ARROW_KEYS = { LEFT: 37, UP: 38, RIGHT: 39, DOWN: 40 };
```

Comment:
I think these two can be combine these key code with the KEY_CODES const since you are using the actual code here and not the string.  I would rename then to ARROW_UP etc though).